volume/store/store.go

```diff
@@ -194,12 +194,14 @@ func (s *VolumeStore) create(name, driverName string, opts map[string]string) (v
 		}
 	}
 
-	logrus.Debugf("Registering new volume reference: driver %q, name %q", driverName, name)
```

Comment:
I.e., the original message wasn't *wrong*, because it showed what name was *specified*

perhaps @vdemeester has thoughts on that; i.e. Is it valuable to have a log message for this?